setup.py

```diff
@@ -35,10 +33,7 @@
       include_package_data=True,
       zip_safe=True,
       install_requires=[
-          'pytz',
-      ],
-      tests_require=[
-          'mock',
+          'backports.zoneinfo; python_version < "3.9"',
```

Comment:
Ah, backports.zoneinfo. Nice, I didn't know of that one. That makes this a lot better.